sunpy/map/mapbase.py

```diff
@@ -2526,7 +2526,7 @@ def plot(self, annotate=True, axes=None, title=True, autoalign=False,
         msg = ('Cannot manually specify {0}, as the norm '
                'already has {0} set. To prevent this error set {0} on '
                '`m.plot_settings["norm"]` or the norm passed to `m.plot`.')
-        if 'norm' in imshow_args:
+        if imshow_args.get('norm', None) is not None:
             norm = imshow_args['norm']
```

Comment:
```suggestion
        if norm := imshow_args.get('norm', None) is not None:
```
Anyone want to approve using the walrus operator here?

Reply:
I approve.